.env.example

```diff
@@ -10,3 +10,6 @@ FIREBASE_PRIVATE_KEY=""
 FIREBASE_PROJECT_ID=""
 FIREBASE_CLIENT_EMAIL=""
 FIREBASE_DATABASE_URL=""
+# With this token the issue command can be used 30 times per minute instead of 10 times per minute.
+# you can get this from https://github.com/settings/tokens
+GITHUB_TOKEN=""
```

Comment:
Could you make a new line at the end of the file. Not sure if we need the comment above as well.